core/parlia_tests/blockchain_eip_3529_test.go

```diff
@@ -0,0 +1,210 @@
+package parlia_tests
```

Comment:
Do we need add header here for Copyright info?

Reply:
I haven't seen any copyright headers in BSC specific files like `parlia.go`.